docs/SalesforceStreaming-streamingsource.md

```diff
@@ -0,0 +1,91 @@
+# Salesforce Streaming Source
+
+
+Description
+-----------
+This source tracks updates in Salesforce sObjects.
+Examples of sObjects are opportunities, contacts, accounts, leads, any custom object, etc.
+
+To track the updates Salesforce PushTopic events are used.
+
+The generation of PushTopic notifications in Salesforce follows this sequence:
+1. Create a PushTopic based on a SOQL query. The PushTopic defines the channel.
+2. Clients subscribe to the channel.
+3. A record is created, updated, deleted, or undeleted (an event occurs). The changes to that record are evaluated.
+4. If the record changes match the criteria of the PushTopic query, a notification is generated by the server and received by the subscribed clients.
+
+The plugin can either create a PushTopic for you, or use an existing one.
```

Comment:
From now on, let's try to avoid "you" in technical documentation. I think this sentence works better as "The plugin can either create a PushTopic for the user, or use an existing one."